Library/Homebrew/env_config.rb

```diff
@@ -55,161 +55,166 @@ module EnvConfig
                      "formula if it or any of its dependencies has a license on this list.",
       },
-      HOMEBREW_FORCE_BREWED_CA_CERTIFICATES:   {
+      HOMEBREW_FORCE_BREWED_CA_CERTIFICATES:     {
         description: "If set, always use a Homebrew-installed `ca-certificates` rather than the system version. " \
                      "Automatically set if the system version is too old.",
         boolean:     true,
       },
-      HOMEBREW_FORCE_BREWED_CURL:              {
+      HOMEBREW_FORCE_BREWED_CURL:                {
         description: "If set, always use a Homebrew-installed `curl`(1) rather than the system version. " \
                      "Automatically set if the system version of `curl` is too old.",
         boolean:     true,
       },
-      HOMEBREW_FORCE_BREWED_GIT:               {
+      HOMEBREW_FORCE_BREWED_GIT:                 {
         description: "If set, always use a Homebrew-installed `git`(1) rather than the system version. " \
                      "Automatically set if the system version of `git` is too old.",
         boolean:     true,
       },
-      HOMEBREW_FORCE_VENDOR_RUBY:              {
+      HOMEBREW_FORCE_VENDOR_RUBY:                {
         description: "If set, always use Homebrew's vendored, relocatable Ruby version even if the system version " \
                      "of Ruby is new enough.",
         boolean:     true,
       },
-      HOMEBREW_GITHUB_API_TOKEN:               {
+      HOMEBREW_GITHUB_API_TOKEN:                 {
         description: "Use this personal access token for the GitHub API, for features such as " \
                      "`brew search`. You can create one at <https://github.com/settings/tokens>. If set, " \
                      "GitHub will allow you a greater number of API requests. For more information, see: " \
                      "<https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting>" \
                      "\n\n    *Note:* Homebrew doesn't require permissions for any of the scopes, but some " \
                      "developer commands may require additional permissions.",
       },
-      HOMEBREW_GITHUB_PACKAGES_TOKEN:          {
+      HOMEBREW_GITHUB_PACKAGES_TOKEN:            {
         description: "Use this GitHub personal access token when accessing the GitHub Packages Registry "\
                      "(where bottles may be stored).",
       },
-      HOMEBREW_DOCKER_REGISTRY_TOKEN:          {
-        description: "Use this bearer token for authenticating with a Docker registry proxying GitHub Packages.",
+      HOMEBREW_DOCKER_REGISTRY_TOKEN:            {
+        description: "Use this bearer token for authenticating with a Docker registry proxying GitHub Packages. " \
+                     "Preferred over HOMEBREW_DOCKER_REGISTRY_TOKEN_BASIC.",
       },
-      HOMEBREW_GITHUB_PACKAGES_USER:           {
+      HOMEBREW_DOCKER_REGISTRY_BASIC_AUTH_TOKEN: {
```

Comment:
Can you move this before `HOMEBREW_DOCKER_REGISTRY_TOKEN` to they are sorted alphabetically?